pkg/importer/dataStream_test.go

```diff
@@ -7,416 +7,404 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
-	"log"
 	"math/rand"
 	"net/http"
+	"net/http/httptest"
 	"net/url"
 	"os"
+	"os/exec"
 	"path/filepath"
 	"reflect"
+	"strconv"
 	"strings"
-	"testing"
-	"time"
 
+	"syscall"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+
+	"kubevirt.io/containerized-data-importer/pkg/common"
 	"kubevirt.io/containerized-data-importer/pkg/image"
+	"kubevirt.io/containerized-data-importer/pkg/util"
 	"kubevirt.io/containerized-data-importer/tests/utils"
 )
 
 const TestImagesDir = "../../tests/images"
 
+var imageDir, _ = filepath.Abs(TestImagesDir)
+var cirrosData, _ = readFile(filepath.Join(imageDir, "cirros-qcow2.img"))
+var tinyCoreData, _ = readFile(filepath.Join(imageDir, "tinyCore.iso"))
+var testfiles = createTestData()
+var stringRdr = strings.NewReader("test data for reader 1")
+var rdrfile, _ = os.Open(filepath.Join(imageDir, "cirros-qcow2.img"))
```

Comment:
I think this var name is too generic since it always contains only a qcow2 reader. Maybe `qcow2Rdr` is more descriptive?

Reply:
Done